core/liblwm2m.h

```diff
@@ -555,6 +567,9 @@ int lwm2m_configure(lwm2m_context_t * contextP, char * endpointName, char * msis
 // create objects for known LWM2M Servers.
 int lwm2m_start(lwm2m_context_t * contextP);
 
+// manage a bootstrap session
+int lwm2m_bootstrap(lwm2m_context_t * contextP);
```

Comment:
Bootstrap should be automatic when starting the client with an empty server list.